doc/contributing.rst

```diff
@@ -21,6 +21,30 @@ of the users who use the package.
 page as the maintainers about changes or enhancements before too much
 coding is done saves everyone time and effort!
 
+Installing developer version
+----------------------------
+
+The standard installation procedure can be found on the
+[MNE website](https://martinos.org/mne/stable/install_mne_python.html)
```

Comment:
you should not use an absolute path for a local page. You can just use `install_mne_python.html` I think

Reply:
We either want it to be viewable on GitHub, as it is now, or render in our doc building (or both!).

If we only care about how it builds on CircleCI, rather than a local link we'd want to use
```
:ref:`install_python_and_mne_python`
```
But my guess is that the absolute path is to make the GitHub view look okay, at the expense of always linking to `martinos` (which will be different from all other links during CircleCI doc build and deployment).

Reply:
I would still use the `:ref:install_python_and_mne_python` even if this means that if you are using **github.io** you remain in **github.io**.